code/scripts/deploy_stage.sh

```diff
@@ -90,19 +90,22 @@ copy_examples() {
     example_name=$(basename "$example")
     # Only copy actual examples
     if [[ "$EXAMPLE_DIRS" == *"$example_name"* ]]; then
-      mkdir -p "$EXAMPLE_DEST$example_name/$SRS_DEST"
-      if [ -d "$example/"SRS/PDF ]; then
-        cp "$example/"SRS/PDF/*.pdf "$EXAMPLE_DEST$example_name/$SRS_DEST"
+      target_srs_dir="./HTML/$EXAMPLE_DEST$example_name/$SRS_DEST"
+      mkdir -p "$target_srs_dir"
+      if [ -d "$example/SRS/PDF" ]; then
+        cp "$example/SRS/PDF/"*.pdf "$target_srs_dir"
       fi
-      if [ -d "$example/"SRS/HTML ]; then
-        cp -r "$example/"SRS/HTML/ "$EXAMPLE_DEST$example_name/$SRS_DEST"
+      if [ -d "$example/SRS/HTML" ]; then
+        cp -r "$example/SRS/HTML/." "$target_srs_dir"
       fi
-      if [ -d "$example/"src ]; then
+      if [ -d "$example/src" ]; then
         mkdir -p "$EXAMPLE_DEST$example_name/$DOX_DEST"
-        for lang in "$example/"src/*; do
+        for lang in "$example/src/"*; do
           lang_name=$(basename "$lang")
-          mkdir -p "$EXAMPLE_DEST$example_name/$DOX_DEST$lang_name"
-          cp -r "$lang/" "$EXAMPLE_DEST$example_name/$DOX_DEST$lang_name/"
+          if [ "$lang_name" != "swift" ]; then
```

Comment:
Having "swift" hard-coded here is odd because it's based on implicit knowledge, but I think this whole file has the same issue. It would be nice if we had "generation receipts" made by each example, containing an itemized (structured/nested) list of things it generated, that we could use in 2 ways: (i) in `drasil-website` to know what things to "post" on the website, and (ii) with a small script that does the necessary copying needed.